src/main/java/com/epam/aidial/core/service/ResourceService.java

```diff
@@ -99,6 +99,7 @@ public ResourceService(Vertx vertx,
         this.cacheExpiration = Duration.ofMillis(cacheExpiration);
         this.compressionMinSize = compressionMinSize;
         this.prefix = prefix;
+        this.resourceQueue = BlobStorageUtil.toStoragePath(prefix, "resource:queue");
```

Comment:
make similar to redisKey -> resource:prefix/queue

Reply:
fixed